3.evaluate_model/scripts/nbconverted/calculate_silhouette_metrics.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Calculate Silhouette Scores
+# 
+# - per phenotype
+# - per feature space
+
+# In[1]:
+
+
+import pathlib
+import numpy as np
+import pandas as pd
+
+from sklearn.decomposition import PCA
+from sklearn.metrics import silhouette_score, silhouette_samples
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
+
+# For consistent Silhouette input space dimensionality
+n_pca_components = 50
```

Comment:
Where does `50` come from? I know in UMAP we do `2` components, what is the difference when is comes to PCA?

Reply:
Also heads up, in the PR comment you say the number of components is `40` but in here it is `50`, recommend confirming which one is correct/most appropriate.

Reply:
ah! Great catch.

In my experience, 50 is more than enough to capture the majority of the variance in the dataset, which is what we're aiming for. It's more or less an arbitrary number